test/fixtures/factories/common.ts

```diff
@@ -1,23 +1,33 @@
 import BigNumber from 'bignumber.js'
 
-import { mainnetAddresses } from '../../addresses'
+import { mainnetAddresses } from '../../addresses/mainnet'
 import { TokenDetails } from '../types/positionDetails'
 
 export const UNISWAP_TEST_SLIPPAGE = new BigNumber(0.25)
 export const SLIPPAGE = new BigNumber(0.05)
 export const MULTIPLE = new BigNumber(2)
 export const EMODE_MULTIPLE = new BigNumber(9.5)
 
-export const USDC: TokenDetails = {
-  symbol: 'USDC' as const,
-  precision: 6,
-  address: mainnetAddresses.USDC,
+export class USDC {
```

Comment:
We have these options...:
1. Turn them into classes so that the address can be configured.
2. Create separate token helpers for different networks
3. Remove address from the token helpers that already exist